etsd/keys/templates/keys/publickey_detail.html

```diff
@@ -9,6 +9,30 @@
 {% block page_content %}
 
 <div class="row">
+  <!-- Modal -->
```

Comment:
Please use the bootstrapmodal.js for displaying the confirmation modal (there's a file in the repository).